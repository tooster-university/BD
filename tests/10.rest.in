{"open": {"database": "student", "login": "app", "password": "qwerty"}}
{"trolls": {"timestamp": 36477619}}
{"protest": {"timestamp": 72468521, "member": 1, "password": "3QK9NG53Z0", "action": 135, "project": 314, "authority": 406}}
{"support": {"timestamp": 123645787, "member": 1, "password": "3QK9NG53Z0", "action": 120, "project": 292, "authority": 450}}
{"votes": {"timestamp": 153889856, "member": 8, "password": "GSY8LQB48"}}
{"downvote": {"timestamp": 164883299, "member": 10, "password": "NH5U88S", "action": 169}}
{"downvote": {"timestamp": 217455145, "member": 1, "password": "3QK9NG53Z0", "action": 142}}
{"protest": {"timestamp": 226499376, "member": 1, "password": "3QK9NG53Z0", "action": 170, "project": 342, "authority": 403}}
{"trolls": {"timestamp": 237094787}}
{"projects": {"timestamp": 248995823, "member": 1, "password": "3QK9NG53Z0"}}
{"downvote": {"timestamp": 301490314, "member": 9, "password": "OEXHTZN6P7", "action": 173}}
{"upvote": {"timestamp": 334446594, "member": 8, "password": "GSY8LQB48", "action": 178}}
{"votes": {"timestamp": 357676947, "member": 10, "password": "NH5U88S"}}
{"trolls": {"timestamp": 375668333}}
