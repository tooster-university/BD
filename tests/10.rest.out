{"status": "OK"}
{"status": "OK", "data": []}
{"status": "OK"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "OK", "data": []}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "ERROR"}
{"status": "OK", "data": []}
